### DISCUSSION TITLE
There is a way to change database without initialize Beanie again?

### DISCUSSION THREAD

**Discussion by Iazzetta** [BeanieODM/beanie#492] [Question] [answered]
There is a way in Document to change database using `set_database` and `set_collection`? I facing problems with `async/await` calls in `init_beanie`, related in this issue: #440. I need to change database without use init_beanie (more than once).

**Comment by roman-right** [comment, marked as answer]
Hi. This is not implemented yet, but I have a plan to add this feature.

**Reply by Iazzetta** [reply]
I have shared a way to update the database without having to initialize beanie again, which has significantly improved performance. I have also added the relevant code to issue #440, but is closed.
